docs/troubleshooting.md

```diff
@@ -87,10 +87,16 @@ This is a tip for advanced users.  An easy way to check if pipx is the problem
 or a package you're trying to install is the problem, is to try installing it
 using `pip`.  For example:
 
+### Unix or macOS
 ```
 python3 -m venv test_venv
-source test_venv/bin/activate
-python3 -m pip install <problem-package>
+test_venv/bin/python3 -m pip install <problem-package>
+```
+
+### Windows
+```
+python -m venv test_venv
+test_venv/Scripts/python -m pip install <problem-package>
```

Comment:
Looks great! I'm not sure, but I think one windows it needs to be `python.exe`?

Reply:
I believe I actually tried on my VM Windows...but now you make me want to make sure and try it again 😆 

I think like regular entry points on windows, you don't need to type the `.exe`, but I will check again.

Reply:
Yes it works.  😄  Also I remember now that I had to change `python3` to `python` on Windows and I found this out by trying these commands in Windows.